packages/toolpad-app/src/apiErrors.ts

```diff
@@ -0,0 +1,17 @@
+import { ValueOf } from './utils/types';
+
+export const API_ERROR_CODES = {
```

Comment:
I'd create a separate file that exports separate consts so that we don't have to pull in this whole object (which will inevitably grow) if you need only one value.

Reply:
It helps with types, but I'll see if I can export consts and still get good typing.